src/views/product-landing/components/overview-cta/index.tsx

```diff
@@ -19,20 +19,22 @@ function OverviewCta({
 					{heading}
 				</h2>
 				<p className={s.body}>{body}</p>
-				<StandaloneLink
-					className={s.cta}
-					text={cta.text}
-					href={cta.url}
-					color="secondary"
-					icon={
-						isAbsoluteUrl(cta.url) ? (
-							<IconExternalLink16 />
-						) : (
-							<IconArrowRight16 />
-						)
-					}
-					iconPosition="trailing"
-				/>
+				{cta ? (
```

Comment:
There's no cta on the designs, but I'm sure we could add one if we wanted this to be consistent.